src/accelerate/accelerator.py

```diff
@@ -126,6 +126,9 @@ class Accelerator:
             accept implementations of `GeneralTracker` for custom trackers, and can be combined with `"all"`.
         logging_dir (`str`, `os.PathLike`, *optional*):
             A path to a directory for storing logs of locally-compatible loggers.
+        downcast_bf16 (`bool`, *optional*):
+            If set to `True` and using `mixed_precision="bf16"` then `torch.float` will become `bfloat16` and
+            `torch.double` will remain `float32` on TPUs.
```

Comment:
Very much against adding a new Accelerator parameter for such a niche feature. It should be in the config only.

Reply:
Okay, will adjust on Monday!

(This should be in the release after this next one anyways)